infra/terraform/environments/dev/main.tf

```diff
@@ -57,6 +57,19 @@ module "user_nfs" {
     performance_mode = "maxIO"
 }
 
+module "user_nfs_softnas" {
+    source = "../../modules/user_nfs_softnas"
+
+    env = "${var.env}"
```

Comment:
This is really more of a constant than a variable, as SoftNAS requires that the role has this name in all environments. I tried to reference the role name from the global resource definition with `${data.terraform_remote_state.base.softnas_role_name}`, but I ran into an issue with Terraform not seeing that output variable when using the terraform `-target` flag to restrict `plan` and `apply` to just this module, so ended up hardcoding it instead.